src/dotnet/Program.cs

```diff
@@ -149,6 +149,8 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null
                         bool skipFirstRunExperience =
                             environmentProvider.GetEnvironmentVariableAsBool("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", false);
 
+                        ReportDotnetHomeUsage(environmentProvider);
```

Comment:
nit: why here? not somewhere close to Reporter.Output.WriteLine("Performance Summary:");? Or else?

It's better to avoid adding stuff to Progam.cs but I don't know a better place

Reply:
I wanted the output to be before command execution, close to where we would report telemetry being used or not, simply because this variable can alter the behavior of the command that's about to run.  It also affects first-run experience, so I'd like it printed before that too.